sequencing/vcf_to_plink/dxapp.json

```diff
@@ -146,7 +146,5 @@
     "whatsNew": "*1.0.0: Cleaned up, added documentation, moved to Ubuntu 14.04;",
```

Comment:
"whatsNew": "*1.0.0: Cleaned up, added documentation, moved to Ubuntu 14.04;", 
->
"whatsNew": "* 1.0.0 First version of the Convert VCF file to PLINK file app."

For generalized first app release log.